prov/efa/src/efa_rma.c

```diff
@@ -37,8 +37,23 @@
 #include <ofi_iov.h>
 #include "efa.h"
 
-static
-ssize_t efa_rma_post_read(struct efa_ep *ep, const struct fi_msg_rma *msg, uint64_t flags)
+
+/*
+ * efa_rma_post_read() will post a read request.
+ *
+ * Input:
+ *     ep: endpoint
+ *     msg: read operation information
+ *     flags: currently no flags is taken
+ *     self_comm: indicate whether the read is toward
+ *                the end point itself. If self_comm is true,
+ *                src_addr should be FI_ADDR_NOTAVAIL
+ * On success return 0,
```

Comment:
Do we really need these? I prefer we keep the style consistent, adding a line to explain `self_comm` is fine.

Reply:
The plan is eventually to have function documentation for every function. I am adding header for functions we touch